test/topology_experimental_raft/test_tablets.py

```diff
@@ -312,3 +312,80 @@ async def test_streaming_is_guarded_by_topology_guard(manager: ManagerClient):
     await manager.api.move_tablet(servers[0].ip_addr, "test", "test", s1_host_id, dst_shard, replica[0], replica[1], tablet_token)
     rows = await cql.run_async("SELECT pk from test.test")
     assert len(list(rows)) == 0
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_table_dropped_during_streaming(manager: ManagerClient):
```

Comment:
Another problem is that this test file contains many long test cases. Note that test cases in the same file cannot be run concurrently.

Please, as a follow-up, consider splitting the file.